python/cuml/dask/naive_bayes/naive_bayes.py

```diff
@@ -29,12 +31,8 @@
 
 from dask.distributed import default_client
 
-from cuml.dask.common.utils import patch_cupy_sparse_serialization
-
-from cuml.dask.common.base import DelayedPredictionMixin
-
 
-class MultinomialNB(DelayedPredictionMixin):
+class MultinomialNB(object):
```

Comment:
FWIW in Python 3 all classes inherit from `object`. So this isn't strictly required. Though it's fine to keep this if you wish.